contrib/win32/openssh/config.vcxproj

```diff
@@ -460,8 +467,9 @@ copy /Y "$(SolutionDir)uninstall-ssh*ps1" "$(OutDir)"
 copy /Y "$(SolutionDir)OpenSSHUtils.ps*1" "$(OutDir)"
 copy /Y "$(SolutionDir)Fix*FilePermissions.ps1" "$(OutDir)"
 copy /Y "$(SolutionDir)sshd_config" "$(OutDir)sshd_config_default"
+copy /Y "$(SolutionDir)moduli" "$(OutDir)moduli"
```

Comment:
Please find out if this file needs to be signed when ship it in desktop release in github. If so, you need to update that package script before next release

Reply:
I think we don't need to sign this file. Users can modify this file when they download it through github release. At that point, signing doesn't make sense?

@manojampalam - what do you think?

Reply:
Discussed with @manojampalam. Leaving it unsigned for github releases just like sshd_config_default file.